research-hub-web/e2e/app.e2e-spec.ts

```diff
@@ -20,6 +20,10 @@ describe('ResearchHub\'s Basic Functionality', () => {
    * Visits the home page and checks it contains the heading 'Welcome to the ResearchHub'.
    */
   it('can display welcome message', async () => {
+    browser.wait(() => {
+      // Test sometimes fail because the element is not yet present, so we wait.
+      return element(by.css('app-root h1')).isPresent();
+    },TIMEOUT_PERIOD);
```

Comment:
Could you refactor this to work in the same way as we've done elsewhere? E.g.:

```typescript
await browser.driver.wait(protractor.ExpectedConditions.visibilityOf(element(by.css('app-root h1'))), TIMEOUT_PERIOD)
```

Reply:
As discussed, we will use browser.* methods which work with Angular from now on. :smile: